src/datalad_installer.py

```diff
@@ -1650,7 +1651,22 @@ def install_package(
             cmd.append(package)
         else:
             cmd.append(f"{package}={version}")
-        runcmd(*cmd)
+        i = 0
+        while True:
+            try:
+                runcmd(*cmd)
+            except subprocess.CalledProcessError as e:
+                if i < 3:
+                    log.error(
+                        "Command failed with exit status %d; sleeping and retrying",
+                        e.returncode,
+                    )
+                    i += 1
+                    sleep(5)
+                else:
+                    raise
+            else:
+                break
```

Comment:
if we run into similar other case , will need try_multiple or alike (and add exponential sleep growth), i guess ok for this only case for now.
Thank you @jwodder